### PR TITLE
Fixed #6759

### LINKED ISSUE #6759: When opened first time on a waypoint, the calculator does not takeover the coordinates correctly
##### Detailed steps causing the problem:
- Add a new waypoint (e.g. for GC78Y21)
- Enter coordinates dialog, press cache coordinates (with GC78Y21 I get N48 31.829 and E11 31.300)
- Open calculator with "Calculate coordinates"

##### Actual behavior after performing these steps:
Calculator opens with different coordinates (here N48 31.049 and E11 31.018)
They differ in the last three digits.

##### Expected behavior after performing these steps:
Calculator should takeover the coordinates (usually current position or cache coordinates) when opened first time for a waypoint.

##### Version of c:geo used:
2017.10.12-NB-7c79739

##### Is the problem reproducible for you?
Yes

##### System information:
```
--- System information ---
Device: D5503 (D5503, Sony)
Android version: 5.1.1
Android build: 14.6.A.1.236
c:geo version: 2017.10.12-NB-7c79739
Google Play services: disabled - 11.3.02 (236-161239932)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: orientation
Hide own/found: true
Map strategy: detailed
HW acceleration: enabled (default state)
System language: de_DE
System date format: dd.MM.yyyy
Debug mode active: no
System internal c:geo dir: /data/data/cgeo.geocaching (2,2 GB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (2,2 GB free) external non-removable
Geocache data: /storage/sdcard1/Android/data/cgeo.geocaching/files/GeocacheData (7,7 GB free) external removable
Database: /storage/emulated/0/Android/data/cgeo.geocaching/files/databases/data (39,7 MB) on user storage
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
   geocaching.com: Logged in (Anmeldung OK) / BASIC
Geocaching.com date format: MM/dd/yyyy
Installed c:geo plugins:  none
--- End of system information ---
```

##### Other comments and remarks:
I am quite sure that this worked before. Probably it sneaked in with the last changes.

### PR DESCRIPTION
**Pull request by S-Bartfast**
" When opened first time on a waypoint, the calculator does not takeover the coordinates correctly" resolved.

### CODE REVIEW

File: main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java [hunk 1]
```diff
@@ -542,6 +542,39 @@ private void displayToast(final int message, final boolean shortToast) {
         }
     }
 
+    private void setButtonInputValuesFromGP()
+    {
```
Comment (pstorch) [inline main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java]: "{" should be on the end of the previous line

**Comment by pstorch** [comment]
add to whitelist

**Comment by pstorch** [comment]
retest this please

**Comment by kumy** [comment]
retest this please

File: main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java (hunk 1 above)
Comment (S-Bartfast) [reply main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java]: Hmm, funny Codacy only reported one (which I did fix).

Just did a RegEx search and yup, there were several, so I've fixed them as well now ;)

**Comment by pstorch** [comment]
@S-Bartfast thanks :+1:

**Pull request merged**